packages/ui/src/SelectField.tsx

```diff
@@ -41,17 +47,33 @@ export type SelectFieldCoreStaticProps = {
   onBlur?: (e: FocusEvent<HTMLElement>) => void
   error?: string
 }
+
 export type SelectFieldCoreDynamicProps<V> =
   | {
       isClearable: true
+      isMulti?: false
       value: SelectFieldOption<V> | null
       onChange: (newValue: SelectFieldOption<V> | null) => void
     }
   | {
       isClearable?: false
+      isMulti?: false
       value: SelectFieldOption<V>
       onChange: (newValue: SelectFieldOption<V>) => void
     }
+  | {
```

Comment:
Should we also use the same technique in `SelectFieldFormikProps`? Could we also have a story for the formik integration of MultiSelect?

Reply:
this is done and committed. i'll request a review specifically for this.